config/routes.rb

```diff
@@ -13,4 +13,10 @@
   resources :users, only: [:index, :show] do
     resources :posts, only: [:index, :show] 
   end
+
```

Comment:
- [ ] I'm unable to create or migrate the database. Here is the error
![Screenshot from 2022-03-03 21-35-57](https://user-images.githubusercontent.com/46388113/156657158-4194d1cb-525e-464a-93c3-082edf7565e8.png)

Reply:
@addod19 ,
Thank you for the feedback and so sorry for the error. The issue has been fixed.

![image](https://user-images.githubusercontent.com/10332499/156661031-fa021736-75b4-4407-9cc2-7246478ae967.png)